terraform/openstack/templates/resources-outputs.tf

```diff
@@ -0,0 +1,84 @@
+output "default_key_name" {
+  value = "${openstack_compute_keypair_v2.bosh.name}"
+}
+
+output "private_key" {
+  value = "${openstack_compute_keypair_v2.bosh.private_key}"
+  sensitive = true
+}
+
+output "external_ip" {
+  value = "${openstack_networking_floatingip_v2.jb.address}"
+}
+
+output "vms_security_groups" {
+  value = ["${openstack_networking_secgroup_v2.vms.name}"]
+}
+
+output "jumpbox__default_security_groups" {
+  value = ["${openstack_networking_secgroup_v2.jb.name}"]
+}
+
+output "director__default_security_groups" {
+  value = ["${openstack_networking_secgroup_v2.bosh.name}"]
+}
+
+output "internal_cidr" {
+  value = "${openstack_networking_subnet_v2.bosh_subnet.cidr}"
+}
+
+output "internal_gw" {
+  value = "${openstack_networking_subnet_v2.bosh_subnet.gateway_ip}"
+}
+
+output "net_id" {
+  value = "${openstack_networking_network_v2.bosh.id}"
+}
+
+output "internal_ip" {
+  value = "${cidrhost(openstack_networking_subnet_v2.bosh_subnet.cidr, 10)}"
+}
+
+output "router_id" {
+  value = "${openstack_networking_router_v2.bosh_router.id}"
+}
+
+output "director__internal_ip" {
+  value = "10.0.1.6"
```

Comment:
I'm not sure if hardcoding this makes sense, you could also just use terraform's `cidrhost` function to select the `nth` IP from a CIDR: https://www.terraform.io/docs/configuration/interpolation.html#cidrhost-iprange-hostnum-

Reply:
Done. Thanks.